AI-gateway-revamp: Update doc (#18397)
mlflow/mlflow: docs/docs/genai/governance/ai-gateway/configuration.mdx

        openai_api_key: $OPENAI_API_KEY
```

### Traffic route

> Configure traffic route that can split incoming traffic to multiple endpoints:

Reviewer: ```suggestion
Add the `routes` configuration to split incoming traffic to multiple endpoints:
```
